shell/platform/android/io/flutter/view/FlutterView.java

```diff
@@ -100,6 +100,10 @@
         int physicalViewInsetRight = 0;
         int physicalViewInsetBottom = 0;
         int physicalViewInsetLeft = 0;
+        int systemGestureInsetTop = 0;
```

Comment:
All changes to this old `FlutterView` probably need to be made to the new `FlutterView`, too. Please see io/flutter/embedding/android/FlutterView.java

Also, it would probably be a good idea to introduce a few `FlutterView` tests around this definition using robolectric.

Reply:
I started exploring the use of roboelectric, but I'm not very familiar with Flutter's android embedding. Do you have any pointers to a high-level overview of how the embedding works and how to get started writing the test suite for a `FlutterViewTest.java`?